tls/s2n_record_write.c

```diff
@@ -191,7 +191,7 @@ static inline int s2n_record_encrypt(
 
 int s2n_record_writev(struct s2n_connection *conn, uint8_t content_type, const struct iovec *in, int in_count, size_t offs, size_t to_write)
 {
-    struct s2n_blob iv;
+    struct s2n_blob iv = { 0 };
```

Comment:
Why not add `notnull_check` to this function (and below).

Reply:
Because Infer didn't complain about this function being called with a potential null value anywhere :) I can add the checks tho.